library/ssl_srv.c

```diff
@@ -2788,6 +2991,14 @@ static int ssl_write_server_hello( mbedtls_ssl_context *ssl )
     ext_len += olen;
 #endif
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+    if ( ssl->conf->transport == MBEDTLS_SSL_TRANSPORT_DATAGRAM )
+    {
```

Comment:
As above: This skipping logic would normally be placed in the handler. Not a blocker.